ext/tags.go

```diff
@@ -19,30 +19,32 @@ var (
 	//////////////////////////////////////////////////////////////////////
 
 	// SpanKind hints at relationship between spans, e.g. client/server
-	SpanKind = spanKindTag("span.kind")
+	SpanKind = spanKindTagName("span.kind")
 
 	// SpanKindRPCClient marks a span representing the client-side of an RPC
 	// or other remote call
-	SpanKindRPCClient = SpanKindEnum("client")
+	SpanKindRPCClientEnum = SpanKindEnum("client")
```

Comment:
The conflation I am referring to is that all of the other variables specify just part of a tag, while these two are also options. And the consistency issue with `Enum` I am having is that it feels like a workaround, so that the label can coexist with the option. Every other string value in this package doesn't have the `Enum` prefix; and `Enum` as a prefix feels inherently unidiomatic.